x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_chart_flyout/datafeed_chart_flyout.tsx

```diff
@@ -510,7 +510,7 @@ export const DatafeedChartFlyout: FC<DatafeedChartFlyoutProps> = ({
                           key={'source-results'}
                           color={euiTheme.euiColorPrimary}
                           id={i18n.translate('xpack.ml.jobsList.datafeedChart.sourceSeriesId', {
-                            defaultMessage: 'Source indices',
+                            defaultMessage: 'Source indices document count',
```

Comment:
IMO we could make these phrases more similar in layout (so the different intent is clearer). For example:
```suggestion
                            defaultMessage: 'Count of documents in source indices',
```